content/docs/authorization/v2.0 Tech Preview/_index.md

```diff
@@ -14,6 +14,8 @@ The following diagram shows a high-level overview of CSM for Authorization with
 
 ![Alt text](image.png)
 
+>> NOTE: Proxy cluster in above picture is a Kubernetes cluster must be owned by the storage administrator.
```

Comment:
Please change to:

">> NOTE: Proxy Cluster in the image above is a Kubernetes cluster.  This cluster must be owned by the storage administrator."